lecture/README.md

```diff
@@ -76,4 +76,3 @@ Then run:
 docker-compose -f src/main/docker/app.yml up -d
 ```
 
```

Comment:
```suggestion
```
To remove Codacy issue

Reply:
I am a bit annoyed why I don't see the Codacy issues :unamused: :smiley: I guess there is a problem with Codacy lately